src/main/java/org/jabref/gui/fieldeditors/TypeEditorViewModel.java

```diff
@@ -23,6 +23,7 @@ public TypeEditorViewModel(Field field, SuggestionProvider<?> suggestionProvider
         itemMap.put("software", Localization.lang("Software"));
         itemMap.put("datacd", Localization.lang("Data CD"));
         itemMap.put("audiocd", Localization.lang("Audio CD"));
+        itemMap.put("inproceedings", Localization.lang("InProceedings"));
```

Comment:
You don't need to add it here. This class is only used for the field `type` in the entry editor, and there are predefined values for this field.

Reply:
Ah, my fault. What code comment should go there to point to the code where this happens? 😇